auto_install/install.sh

```diff
@@ -997,94 +1017,151 @@ installPiVPN(){
 		confOVPN
 		confNetwork
 		confLogging
+
 	elif [ "$VPN" = "wireguard" ]; then
+
+		# Since WireGuard only uses UDP, askCustomProto() is never called so we
+		# set the protocol here.
+		pivpnPROTO="udp"
+		pivpnDEV="wg0"
+		pivpnNET="10.6.0.0"
+		vpnGw="${pivpnNET/.0.0/.0.1}"
+
 		installWireGuard
 		askCustomPort
 		askClientDNS
 		askPublicIPOrDNS
 		confWireGuard
 		confNetwork
+
+		echo "pivpnPROTO=${pivpnPROTO}" >> /tmp/setupVars.conf
+
 	fi
+
+	echo "pivpnDEV=${pivpnDEV}" >> /tmp/setupVars.conf
+	echo "pivpnNET=${pivpnNET}" >> /tmp/setupVars.conf
+	echo "subnetClass=${subnetClass}" >> /tmp/setupVars.conf
 }
 
 askWhichVPN(){
 	if [ "${runUnattended}" = 'true' ]; then
-		if [ -z "$VPN" ]; then
-			echo ":: No VPN protocol specified, using WireGuard"
-			VPN="wireguard"
-		else
-			VPN="${VPN,,}"
-			if [ "$VPN" = "wireguard" ]; then
-				echo "::: WireGuard will be installed"
-			elif [ "$VPN" = "openvpn" ]; then
-				echo "::: OpenVPN will be installed"
+		if [ "$PLAT" = "Raspbian" ] || [ "$X86_SYSTEM" -eq 1 ]; then
+			if [ -z "$VPN" ]; then
+				echo ":: No VPN protocol specified, using WireGuard"
+				VPN="wireguard"
 			else
-				echo ":: $VPN is not a supported VPN protocol, please specify 'wireguard' or 'openvpn'"
-				exit 1
+				VPN="${VPN,,}"
+				if [ "$VPN" = "wireguard" ]; then
+					echo "::: WireGuard will be installed"
+				elif [ "$VPN" = "openvpn" ]; then
+					echo "::: OpenVPN will be installed"
+				else
+					echo ":: $VPN is not a supported VPN protocol, please specify 'wireguard' or 'openvpn'"
+					exit 1
+				fi
+			fi
+		elif [ "$X86_SYSTEM" -eq 0 ]; then
```

Comment:
The above line could/should be replaced with `else`, since this must be true if `[ "$PLAT" = "Raspbian" ] || [ "$X86_SYSTEM" -eq 1 ]` is false? Otherwise if it is possible that both is false (e.g. $X86_SYSTEM == 2), then VPN would not be assigned, which will make the installer run into issues later?

Same should be true for line 1090 below.

Reply:
@orazioedoardo 
Yeah it practically does not matter. I personally just like to see all theoretical cases handled, even if they practically cannot occur 😉.